Description: Add missing include needed for building with Qt 6 on Jammy
Author: Peter J. Mello <admin@petermello.net>
Origin: other, https://git.launchpad.net/~libretro/libretro/+git/RetroArch-debian/commit/patches/0005-add-qhash-include.patch?id=7ed2946c2a08c4038f9f29ba05963a5bf56b2073
Forwarded: no
Last-Update: 2025-06-26

--- a/ui/drivers/qt/qt_dialogs.h
+++ b/ui/drivers/qt/qt_dialogs.h
@@ -3,6 +3,7 @@
 
 #include <QDialog>
 #include <QPointer>
+#include <QHash>
 
 #include <retro_common_api.h>
 
